libs/shared/feature/form-nuv/src/lib/components/advanced/address/formly/formly-address.component.ts

```diff
@@ -15,22 +15,22 @@ import { FormlyAddressFieldProperties } from './formly-address.model';
 })
 export class FormlyAddressComponent extends FormlyBaseComponent<FormlyAddressFieldProperties> implements FormlyExtension, OnInit {
   public countrySelectLabels = new Map();
+  public reviewDetailsMap = new Map<string, string>();
```

Comment:
Maybe we should implement better type safety here, then we can maintain the typesafety in the template and continue to use the object.property syntax
```suggestion
  public reviewDetailsMap = new Map<IFieldKeys, string>();
```

Reply:
Yes that would be a new interface